### PR TITLE
Nuxt Devtools integration

### PR DESCRIPTION
**Pull request by markus-gx**
Added basic integration for Storyblok inside nuxt-devtools. 

image

### CODE REVIEW

**Review by Dawntraoz — APPROVED** [review]
What an excellent idea @markus-gx 😍

**Comment by markus-gx** [comment]
> What an excellent idea @markus-gx 😍

Possibly enhancements in the feature could be to automatically show the corresponding space/page inside Storyblok without the need to navigate in an iframe. But Devtools is not yet capable to do that (Adjusting iFrame Routes based on current route on client side). 🤔

File: package.json [hunk 1]
```diff
@@ -15,6 +15,7 @@
   "devDependencies": {
     "@commitlint/cli": "^17.4.4",
     "@commitlint/config-conventional": "^17.4.4",
+    "@nuxt/devtools": "^0.2.5",
```
Comment (alexjoverm) [inline package.json:18]: @markus-gx I believe this devDep should go into playground/package.json

File: README.md [hunk 2]
```diff
@@ -77,6 +77,7 @@ export default defineNuxtConfig({
 #### Options
 
 When you initialize the module, you can pass all [_@storyblok/vue_ options](https://github.com/storyblok/storyblok-vue#storyblok-api) plus a `bridge` option explained in our [JS SDK Storyblok bridge section](https://github.com/storyblok/storyblok-js#storyblok-bridge).
+If you want to use Storyblok inside `nuxt-devtools` you can use the option `devtools`.
```
Comment (alexjoverm) [inline README.md:80]: @markus-gx can we add a note on the lines of "if you enable the devtools option, make sure to have installed the `@nuxt/devtools` module and enable it on your nuxt config"

If you feel like you need extra space, we can create a small h3 section for the devtools

**Review by alexjoverm — CHANGES_REQUESTED** [review]
@markus-gx I love your creativity, so cool DX :) Added couple of minor comments and we're good to go

**Review by alexjoverm — APPROVED** [review]

**Comment by alexjoverm** [comment]
@markus-gx I approved it, but it has some conflicts to resolve. I think if you merge latest `main` should be fixed. As soon as you push it, we'll merge! Really cool stuff, thank you

**Comment by alexjoverm** [comment]
Amazing! Kudos @markus-gx

**Comment by samuells** [comment]
👏 👏 👏  thank you @markus-gx

**Pull request merged**